manuscript/applications.tex

```diff
@@ -0,0 +1,36 @@
+\section{Other Uses of Dynamic Deployment}
+The concept of dynamic demand-driven deployment is not new,
+and has been used in various applications. Various algorithms
+are used to optimize deployment of facility or utilization
+of given sources.
+
+\subsection{Aircraft Dispatch}
+To maximize fleet utilization and minimize
+operating costs, airlines predict future demands
+and optimize their flight schedule and aircraft
+type using linear programming methods \cite{berge_demand_1993}. The Demand
+Driven Dispatch process evaluates benefits and costs
```

Comment:
Introduce "The Demand Driven Dispatch" process in a manner that recognizes that the reader has never heard of it. 
Something like : 
```
Benefits and costs of ticket pricing, aircraft assignment, and crew scheduling are evaluated with a linear optimization method called ``Demand Driven Dispatch'' \cite{shebalov_practical_2009}, a type of deterministic optimization method.
```